frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/datatables/data/PaginationParameters.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Siemens AG, 2018. Part of the SW360 Portal Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.sw360.portal.common.datatables.data;
+
+/**
+ * @author thomas.maier@evosoft.com
+ */
+public class PaginationParameters {
+
+    private String echo;
+    private String columnNames;
+    private int displayLength;
+    private int displayStart;
+    private int columnCount;
+    private int sortingColumn;
+    private boolean ascending;
+
+    public PaginationParameters() {
+    }
+
+    public PaginationParameters(String echo, String columnNames, int displayLength, int displayStart, int columnCount, int sortingColumn, boolean ascending) {
+        this.echo = echo;
+        this.columnNames = columnNames;
+        this.displayLength = displayLength;
+        this.displayStart = displayStart;
+        this.columnCount = columnCount;
+        this.sortingColumn = sortingColumn;
+        this.ascending = ascending;
+    }
+
+    public String getEcho() {
```

Comment:
From the variable name it is not clear what echo should be used for. Maybe add some documentation?

Reply:
added documentation for the pagination parameter fields